examples/grafana_dashboard/main.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    coralogix = {
+      version = "~> 1.3"
+      source  = "coralogix/coralogix"
+    }
+  }
+}
+
+provider "coralogix" {
+  #api_key = "<add your api key here or add env variable CORALOGIX_API_KEY>"
+  #env = "<add the environment you want to work at or add env variable CORALOGIX_ENV>"
+}
+
+resource "coralogix_grafana_dashboard" dashboard {
```

Comment:
This is different then what we discussed.
We wanted hosted_dashboard and that should be used for both grafana and opensearch